ReservePubKey/__tests__/handler.test.ts

```diff
@@ -107,15 +118,7 @@ describe("reserveSingleKey", () => {
       pubKey.pub_key
     )(assertionRef)();
 
-    expect(loggerMock.trackEvent).toHaveBeenCalledWith({
-      name: "lollipop.error.reserve-pubkey",
-      properties: {
-        message: `${FN_LOG_NAME} | COSMOS_ERROR_RESPONSE`,
-      },
-      tagOverrides: {
-        samplingEnabled: "false"
-      }
-    });
+    expect(loggerMock.trackEvent).toHaveBeenCalledTimes(0);
```

Comment:
Add a small comment that explain that no event is sent by the `reserveSingleKey` function

Reply:
Done in [a14d8d7](https://github.com/pagopa/io-functions-lollipop/pull/36/commits/a14d8d78391a5f98b6d525db50d3b2db0e393a74)